defaults/main.yml

```diff
@@ -1,7 +1,10 @@
+mysql_hardening_enabled: yes
 # general configuration
 mysql_hardening_user: 'mysql'
+mysql_hardening_group: 'mysql'
```

Comment:
this is new variable that set up a group, who has write access to conf.d folder and  its subfiles